distribution/docs/src/main/resources/content/_configuring/banner-markings-content-extractor.adoc

```diff
@@ -0,0 +1,89 @@
+:title: Banner Markings Extractor
+:type: configuringConfigFile
+:status: published
+:summary: Banner Markings Extractor
+:order: 132
+
+=== Banner Markings Extractor
+
+The Banner Markings Extractor parses the text of PDFs, Microsoft Office documents, plain text files,
+or any other file from which the <<_tika_input_transformer,TikaTransformer>> can extract text.
+The ${branding} default behavior is to apply high-water security settings when ingesting a document.
+This is sufficient for documents with no security markings, but is incorrect when working with
+classified documents that include security mark-up in the text.
+
+The Banner Markings Extractor parses a banner marking and attempts to create new attributes on the
```

Comment:
❗️ We should probably be more clear about the distinction between attributes and attribute values, and note things like adding the security attributes to an injector definition so they all get stored.

Reply:
I remember the plugin creating attribute descriptors. I don't remember the attribute descriptors getting defined and inject-- I think they are created in code.

I'll see what I can do about differentiating attributes from their values.